src/vm/docs/mod.rs

```diff
@@ -737,6 +747,114 @@ definition (i.e., you cannot put a define statement in the middle of a function
 "
 };
 
+const MINT_TOKEN: SpecialAPI = SpecialAPI {
+    name: "mint-token!",
+    input_type: "TokenName, int, principal",
+    output_type: "(response bool int)",
+    signature: "(mint-token! token-name amount recipient)",
+    description: "`mint-token!` is used to increase the token balance for the `recipient` principal for a token
+type defined using `define-token`. The increased token balance is _not_ transfered from another principal, but
+rather minted.",
+    example: "
+(define-token stackaroo)
+(mint-token! stackaroo 100 tx-sender)
+"
+};
+
+const MINT_ASSET: SpecialAPI = SpecialAPI {
+    name: "mint-asset!",
+    input_type: "AssetName, A, principal",
+    output_type: "(response bool int)",
+    signature: "(mint-asset! asset-class asset-identifier recipient)",
+    description: "`mint-asset!` is used to instantiate an asset and set that asset's owner to the `recipient` principal.
+The asset must have been defined using `define-asset`, and the supplied `asset-identifier` must be of the same type specified in
+that definition.
+
+If an asset identified by `asset-identifier` _already exists_, this function will return an error with the following error code:
+
+`(err 1)`
```

Comment:
Should we standardize (or have a plan to standardize) error codes, if this is the approach we're going to take?

Reply:
I have no real opinion either way. As long as the error codes are documented, I'm not sure if standardizing is necessary.